### DISCUSSION TITLE
Settings have lost a panel?

### DISCUSSION THREAD

**Discussion by esksp** [endless-sky/endless-sky#10721] [General]
I thought there was a "Next" button on one of the Settings panels, but I wasn't sure I remembered right.
Today I wanted to check the hotkey for flotsam collection, and didn't find it anymore. There is truly a panel missing:

![ScreenshotControls](https://github.com/user-attachments/assets/403da328-ae64-47de-bc6f-d0ae4c2c0048)

Is this a bug, or did I overlook something?

--- Edit:
BTW, this already happened in 0.10.8, I am using 0.10.10 now.

**Comment by TomGoodIdea** [comment]
What plugins do you have?

**Reply by esksp** [reply]
Thanks! That was it.

I have a small plugin which changes the flagship "shields" ring color (see #10595): I just took the original "interfaces text" file and changed the RGB value in one line. I don't know what went wrong and why, but now I took the original file, redid my `color "overlay flagship shields" 1. 1. 1. .25` modification, and now the "Next" button is back.

**Reply by TomGoodIdea** [reply]
You don't need to copy the whole file. Redefine only what you need and you'll be less likely to run into such problems.

**Reply by esksp** [reply]
> Redefine only what you need

Yes, but I don't know the exact syntax, i.e. if the game replaces things on file level or on individual setting level.

For instance for adding any new system links it apparently needs an "add" in front.
I don't know if there is some secret command for _that_ kind of file too, and I was afraid that making a new "interfaces text" with only the modified  line could cause the game to only use the almost empty "interfaces text" and disregard the complete one...
So I copied the whole file, just in case. Usually I'm pretty good at copy/paste... :-D
Oh well.

**Reply by danaris** [reply]
That's what Tom is telling you: the game replaces individual elements, so you don't need the whole file. 

The way the game is designed, you could just put _all_ the data in one big text file and it would just load it all in happily. There is nothing special about any specific file or file name. All the specification happens at the top-level element level (ie, the unindented lines define what comes in the indented section following, if any).

**Reply by esksp** [reply]
> That's what Tom is telling you

I know, I know, I was just telling him that I didn't know that, and didn't wanted to take the risk.

The point being (as usually with me), that the Wiki should be updated/completed... It would probably reduce the amount of stupid newbie questions like mine...  :-)
